tests/unit_tests/test_stress.py

```diff
@@ -25,6 +27,8 @@ def compile_test():
     run(command, check=True, shell=True)
 
 
+# noinspection PyUnresolvedReferences
+@pytest.mark.skip(reason="broken after new implementation")
```

Comment:
It looks like `test_stress` and `test_parallel` are broken bcs of `bpe.cpp` fix. Please verify that they can be skipped.
@SokolovYaroslav You can find test output at `Test YouTokenToMe` build config in pinned builds

Reply:
That is expected, yes